### PR TITLE
suggested reading reusable component done

### LINKED ISSUE #4326: Make a reusable component for "suggested reading" section.
#### Current State
As you can see this `suggested reading` section is used in most of the docs pages. This section's code is repeated for every such page.  

#### Desired State
A reusable `suggested reading` component is to be made which could be used for all the pages.
---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by shubham-tyagi**
Component for suggested reading section has been made and and has been placed at the correct
position in all the required files.

Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**

This PR fixes #4326

**Notes for Reviewers**

Please point out if I missed something.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md [hunk 1]
```diff
@@ -11,27 +11,26 @@ To authenticate with Meshery through `mesheryctl` you will use the command `mesh
...
-        <div class="codeblock">
-        mesheryctl system config --token [path-to-file]
-        </div>
-    </pre>
-<br />
-
+  <br/>
+  > To get the token through `mesheryctl` you would have to use the following command and the path to token for authenticating to Meshery API (default "auth.json").
+      <br/>
+      <pre class="codeblock-pre">
```
Comment (warunicorn19) [inline docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md:28]: hey @shubham-tyagi could you please check what's happening in this part, this is supposed to be a code block, but it's appearing in form of normal text.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Everything looks good to me, just 1 minor change, Great stuff! @shubham-tyagi

File: docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md (hunk 1 above)
Comment (shubham-tyagi) [reply docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md:28]: thanks for reviewing @warunicorn19.  Are there any same issues anywhere else or just this?

Comment (warunicorn19) [reply docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md:28]: I couldn't find anymore 😅

Comment (shubham-tyagi) [reply docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md]: fixed the codeblock issue @warunicorn19

**Comment by l5io** [comment]
🚀 Preview for commit 978ef3dfcdf20200757cf05a808ac6e4d7bb639e at: https://615df73d1596d000a71426f1--meshery-docs-preview.netlify.app

**Comment by shubham-tyagi** [comment]
can you also add the "hactoberfest-accepted" label on this issue @warunicorn19

**Comment by warunicorn19** [comment]
I know, I just do it for fun 😂😂

**Pull request merged**